rust/agents/scraper/src/agent.rs

```diff
@@ -195,7 +195,7 @@ impl Scraper {
             .await
             .unwrap();
         let cursor = sync.cursor(index_settings.clone()).await;
-        tokio::spawn(async move { sync.sync("message_dispatch", cursor).await }).instrument(
+        tokio::spawn(async move { sync.sync("message_dispatch", cursor.into()).await }).instrument(
```

Comment:
I know we didn't chat about this but imo it makes sense to also move the scraper and validator to using hook indexing, wdyt?

Reply:
we can't enable hook indexing for the validator bc we don't have a message indexer, but as discussed it's probably fine without because the checkpoint submitter already uses sequences.

I did enable it for the scraper 👍